e2e-tests/common/constants.ts

```
@@ -16,9 +18,8 @@ export enum RedirectToLegacy {
   both = "both",
 }
```

The config variables below are now read in the same way as svelte config vars.  Additionally, if an illegal value is passed to `REDIRECT_TO_LEGACY` an error is thrown rather than silently reverting to the default.